man/docker-inspect.1.md

```diff
@@ -201,8 +201,8 @@ output:
       {{$p}} -> {{(index $conf 0).HostPort}} {{end}}' d2cc496561d6
       80/tcp -> 80
 
-You can get more information about how to write a go template from:
-http://golang.org/pkg/text/template/.
+You can get more information about how to write a Go template from:
+http://golang.org/pkg/text/template.
```

Comment:
Looks like the canonical URL is including the slash - the site redirects to; `http://golang.org/pkg/text/template/`. So to prevent a redirect, I think we should keep the trailing slash.

It looks like the site is also reachable on `https://`, so could you change it to `https://golang.org/pkg/text/template/`?